FuelCell/.vscode/tasks.json

```diff
@@ -0,0 +1,25 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+        {
+			"label": "Debug Android Build",
+			"type": "shell",
+			"command": "dotnet build -p:Configuration=Debug ${workspaceFolder}/FuelCell.Android/FuelCell.Android.csproj \"/t:Install;_Run\" /p:AndroidAttachDebugger=true /p:AndroidSdbHostPort=10000",
```

Comment:
@CartBlanche personally I would split this up into a task for `Building` and one for `Installing/Running`.
If someone calls this to just build the app by Shift+Cmd+B then it will fail if a device is not attached.

Reply:
@dellis1972 If they press Shift+Ctrl+D and it goes through the Build/Deploy they get a very clear error:
```
C:\Program Files\dotnet\packs\Microsoft.Android.Sdk.Windows\34.0.85\tools\Xamarin.Android.C 
ommon.Debugging.targets(678,5): **error XA0010: No available device**. [C:\Users\savag\source\r 
epos\CartBlanche\MonoGame\MonoGame.Samples\FuelCell\FuelCell.Android\FuelCell.Android.cspro 
j]
    1 Warning(s)
    1 Error(s)
```
So I only see a marginal benefit to adding a separate Build config. The docs should maybe make it clear that for Android and iOS if no device or simulator is running then debugging can't proceed. I also don't think many people will want to build the app, but not see it running. If they don't have a sim/device, I suspect they just won't try it.

Reply:
its a preference thing I guess. I do local builds all the time during development, I only actually run up the app when I need to test it 🤷

Reply:
@dellis1972 this will still build, so they don't lose that ability and they just get the deployment error message at the end.